packit/schema.py

```diff
@@ -169,6 +162,17 @@ def load(self, *args, **kwargs):
             result = super().load(*args, **kwargs).data
         return result
 
+    def dump(self, *args, **kwargs):
+        if MM3:
+            result = super().dump(*args, **kwargs)
+        else:  # v2
+            result = super().dump(*args, **kwargs).data
+        return result
+
+    @post_dump
+    def remove_none_values(self, data, **kwargs):
+        return {key: value for key, value in data.items() if value}
```

Comment:
this will remove also 0 and empty string and based on function name only None should be removed.

```suggestion
        return {key: value for key, value in data.items() if value is not None}
```

Reply:
thanks, changed